src/GanttChart.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module GanttChart where
+
+import Control.Lens.TH (makeLenses)
+import Parser
+import Text.Megaparsec as M
+import Text.Megaparsec.Char (alphaNumChar, char, string)
+
+data GanttChartGraph = GanttChartGraph
+  { _ganttChartTitle :: GanttChartTitle,
+    _dateFormat :: DateFormat,
+    _axisFormat :: AxisFormat,
+    _section :: [Section]
+  }
+  deriving (Eq, Show, Generic)
+
+data Section = Section
+  { _sectionTitle :: SectionTitle,
+    _tasks :: [Task]
```

Comment:
Need to be consistent with record field naming.

Reply:
For eg., `_gcgTitle` above and `_sectionTitle` and `_sectionTasks` here. More generally, `_<type><Field>`.

Reply:
Just makes the lens use a bit more obvious and clear.